src/main/java/me/bariahq/bariahqmod/command/Command_staffchat.java

```diff
@@ -13,12 +16,24 @@
 
     @Override
     public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
-    {
+    {FPlayer player = plugin.pl.getPlayer(playerSender);
         if (args.length > 0)
         {
             plugin.cm.adminChat(sender, StringUtils.join(args, " "));
             return true;
+        } else {
+            if (!player.inAdminChat()){
+                player.setAdminChat(true);
+                playerSender.sendMessage(ChatColor.AQUA + "Staff chat enabled.");
+                return true;
+            } else if (player.inAdminChat()){
```

Comment:
Can use else statement or something similar to msg(!adminChatToggled ? “Disabled” : “Enabled”);
On my phone atm.